README.md

```diff
@@ -1 +1,73 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Capstone
+
+> in this project:
+- Set up the linters for html, css, and JavaScript.
+- Create a home and about pages
+- Add cards dynamically through JavaScript
+
+
+## Built With 
+
+- Html
+- Css
+- JavaScript
+- VS Studio
+- Git Bash
+
+## Live Demo (if available)
+
+[Click](https://ibr5500.github.io/Capstone/) for Live Demo Link
+
+
+### Prerequisites
+- HTML & CSS & JS
+
+### Setup
+- linters for HTML, CSS and JS
+
+### Run tests
+- Open your terminal, move to the directory where you want to clone the app (i.e cd Documents) 
+- Enter git clone git@github.com:ibr5500.github.io/Capstone/
+- Change to the directory you just clone into (i.e cd Capstone)
+- Open index.html from the list of files in your left handside
+- Right click on your opened index.html file and launch the live server to preview
+- Include any possible changes and run "git add ." 
+- Commit the changes with a descriptive message (i.e xyz features added) 
+- Push the changes to changes to the repository (i.e git push)
+
+### Deployment
+- This app is deployed using Github pages (https://pages.github.com/)
+
+## Author
+
+👤 **Ibrahim Ahmat**
+
+- GitHub: [@ibr5500](https://github.com/ibr5500)
+- Twitter: [@ibr_ahmat](https://twitter.com/ibr_ahmat)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Please give a ⭐️ if you like this project!
+
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration 
+- etc
+
```

Comment:
- [ ] Kindly give appropriate credit to the author [Cindy Shin](https://www.behance.net/adagio07) in the README of your project.